tsconfig.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
this... could be cleaner

Reply:
It could. That was the way the formatter made it. I'll kick it.